pennylane/devices/default_gaussian.py

```diff
@@ -728,9 +728,9 @@ def apply(self, operation, wires, par):
 
```

Comment:
There seems to be a message on line 723-724 where the mention of cov and vector of means can be swapped, maybe good to adjust that too: (cannot make a suggestion)
```python
                    "GaussianState covariance matrix or means vector is "
                    "the incorrect size for the number of subsystems."
```